xl-deploy/8.6/centos/Dockerfile

```diff
@@ -86,16 +86,18 @@ WORKDIR ${APP_HOME}
 RUN useradd -r -M -u 10001 -g 0 xebialabs
 USER 10001
 
-VOLUME ["${APP_HOME}/conf", "${APP_HOME}/hotfix/lib", "${APP_HOME}/hotfix/plugins", "${APP_HOME}/ext", "${APP_HOME}/plugins", "${APP_HOME}/repository", "${APP_HOME}/work"]
+VOLUME ["${APP_HOME}/conf", "${APP_HOME}/hotfix/lib", "${APP_HOME}/hotfix/plugins", "${APP_HOME}/ext", "${APP_HOME}/plugins", "${APP_HOME}/work"]
```

Comment:
why war the repository volume removed?

Reply:
@eltonkevani I think it can be added back, but I think we should discourage users to use it: better to use an external database.

Reply:
yes for prod definitely it should be a DB but for local installation (trial,poc) in either docker or local k8s that should be still there

Reply:
@eltonkevani, @hierynomus reverted. Can you merge this branch and also its parent DEPL-13566 to master, please?